DESIGN DECISIONS.md

```diff
@@ -0,0 +1,21 @@
+# Design decisions
+
+The purpose of this document is to document why certain choices have been made.
+
+## Technology choices
+
+### Lodash templates
+
+To improve the readability and maintainability of our HTML, we've chosen to use lodash templates. These can be found in the [templates folder](templates).
+
+The templates are precompiled using [lodash-cli][lodash-cli], this means the HTML is compiled into javascript. To see this in action look at the [compiled template.js file](js/templates.js).
+
+### Browserify
```

Comment:
Are there alternatives to using Browserify when choosing to implement the module pattern we wish to use? If not, then this section doesn't really add anything and can better be replaced by a few lines about our take on module pattern.